content/sensu-go/5.21/release-notes.md

```diff
@@ -63,6 +64,34 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.21.0 release notes
+
+**June 9, 2020** &mdash; The latest release of Sensu Go, version 5.21.0, is now available for download.
+
+**RELEASE SYNOPSIS NEEDED HERE**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.21.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][158] Added [entity count and limit][156] for each entity class in the tabular title in the response for `sensuctl license info` (in addition to the total entity count and limit).
+- Added `sensuctl user hash-password` command to generate password hashes. The `user` resource now includes a `password_hash` field to specify the password hash instead of the cleartext password.
```

Comment:
```suggestion
- ([Commercial feature][158]) Added OpenSSL-linked binaries for the Sensu agent and backend, with accompanying `--require-openssl` and `--require-fips` flags for the agent and backend.
- Added `sensuctl user hash-password` command to generate password hashes. The `user` resource now includes a `password_hash` field to specify the password hash instead of the cleartext password.
```